recipes/pygeoogc/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pygeoogc" %}
```

Comment:
Please don't modify files outside of the recipe dir you are working on.  Maybe you need to pull from master?

Reply:
wrong spot, sorry